_translator-files/po/de/Tips-Tricks-More.po

```diff
@@ -45,116 +47,118 @@ msgid ""
 "# Tips & Tricks\n"
 " {:.no_toc}\n"
 msgstr ""
+"# Tipps & Tricks\n"
+" {:.no_toc}\n"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:12
 msgid "<details markdown=\"1\">"
-msgstr ""
+msgstr "<details markdown=\"1\">"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:14
 msgid "<summary>Table of contents</summary>"
-msgstr ""
+msgstr "<summary>Inhaltsverzeichnis</summary>"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:16
 msgid "TOC"
-msgstr ""
+msgstr "TOC"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:17
 msgid "{:toc}"
-msgstr ""
+msgstr "{:toc}"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:19
 msgid "</details>"
-msgstr ""
+msgstr "</details>"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:21
 msgid "### Learning about remote band rehearsing"
-msgstr ""
+msgstr "### Band Proben aus der Ferne"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:23
 msgid "Jamulus user [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) has compiled a massive amount of information relating to [Remote Band Rehearsals](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc), which covers topics such as hardware and software configuration including examples and advice for newcomers to the field. He also includes a section on Jamulus in comparison to other solutions."
-msgstr ""
+msgstr "Jamulus-Benutzer [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) hat eine große Menge an Informationen zu [Band Proben aus der Ferne](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) (Google doc) zusammengestellt, die Themen wie Hardware- und Softwarekonfiguration einschließlich Beispielen und Ratschlägen für Neulinge auf diesem Gebiet abdecken. Er enthält auch einen Abschnitt über Jamulus im Vergleich zu anderen Lösungen."
```

Comment:
> Er enthält auch einen Abschnitt über Jamulus im Vergleich zu anderen Lösungen."

**Das** Google Dokument is not male, so I think it should be "es"

Reply:
Corrected